src/components/form/skill/AddSkill.js

```diff
@@ -127,9 +125,13 @@ const AddSkill = ({ onClose, isEdit, values }) => {
 
   const getInitialValues = () => {
     if (isEdit) {
+      const skills = data.get.skills;
+
+      const index = skills.findIndex(skill => skill.id === values.id);
```

Comment:
we can use only find to get us the object

Reply:
Noted and done : [database-integrate 46f9fd5] use find to locate object